ntbs-service/Pages/Alerts/TransferRequestAction.cshtml

```diff
@@ -51,14 +51,35 @@
                 <div class="nhsuk-radios govuk-radios--conditional" data-module="govuk-radios">
                     <div class="nhsuk-radios__item">
                         <input asp-for="AcceptTransfer" nhs-input-type="Radios" type="radio"
-                            value="true" id="accept-transfer-input" aria-describedby="action-error">
+                            value="true" id="accept-transfer-input" data-aria-controls="conditional-radio-accept" aria-describedby="action-error">
                         <label nhs-label-type="Radios" for="accept-transfer-input">
                             Accept transfer
                         </label>
                     </div>
+                    
+                    <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-radio-accept">
+                        <nhs-grid-row>
+                            <nhs-grid-column grid-column-width="OneHalf">
+                                <label asp-for="TargetHospitalId" nhs-label-type="Standard">
+                                    Hospital
+                                </label>
+                                <select nhs-select-type="Standard" asp-items="Model.Hospitals" asp-for="TargetHospitalId">

```

Comment:
What will be the default value for this select list? Is this required field?

Reply:
The default value is the first value in the list, it's required so there isn't a "null" value

Reply:
@jasiekmiko any preference on defaulting to first value in the list of hospitals or to have "select a hospital" as an option and validation to make sure that a hospital is selected? For reference ~50% of the time there will only be one hospital in the list